src/resources/components/Chart.vue

```diff
@@ -1,654 +1,660 @@
-										borderDash: [2, 2],
-										lineWidth: 1,
-										color: 'rgba(233,238,242,1)',
-										zeroLineColor: 'rgba(233,238,242,1)',
-										zeroLineWidth: 1,
-									},
-									ticks: this.ticks,
-								}],
-							xAxes: [
-								{
-									display: true,
-									gridLines: {
-										lineWidth: 0,
-										drawBorder: false,
-									},
-									ticks: this.ticks,
-								}],
-						},
-					};
-
-					if (this.isDateSeriesOverlay) {
-						//options.scales.xAxes[0].type = 'time';
-
-						if (this.yAxisIsCurrency) {
-							// format y axis as currency
-							options.scales.yAxes[0].ticks.callback = function(value, index, values) {
-								if (typeof value == 'number') {
-									return self.formatCurrencyValue(value);
-								}
-
-								return value;
-							};
-
-							/**
-							 * Take a combined label like `2019-07-13#2018-07-13`, get the first date,
-							 * and reduce it to a concise month and day: `7/13`. Used in tooltip titles
-							 * and x axis ticks.
-							 */
-							options.scales.xAxes[0].ticks.callback = function(value, index, values) {
-								return self.getFormattedDateFromCombinedLabel(value);
-							};
-
-							/**
-							 * Remove tooltip's title.
-							 */
-							options.tooltips.callbacks.title = function(item) {
-								return '';
-							};
-
-							/**
-							 * Prepend date for each tooltip value.
-							 */
-							options.tooltips.callbacks.label = function(item, data) {
-								if (item.datasetIndex === 0) {
-									return `${item.label}: ${self.formatCurrencyValue(item.value)}`;
-								}
-
-								const label = data.labels[item.index];
-								const dateLabel = self.getFormattedDateFromCombinedLabel(label, 1);
-
-								return `${dateLabel}: ${self.formatCurrencyValue(item.value)}`;
-							};
-						}
-					}
-
-					return options;
-				} else if (this.type === 'sparkline') {
-					return {
-						responsive: true,
-						animation: false,
-						aspectRatio: this.getAspectRatio(),
-						legend: {display: false},
-						elements: {
-							line: {
-								borderColor: '#d3a87a',
-								borderWidth: 2,
-							},
-							point: {radius: 0},
-						},
-						tooltips: {enabled: false},
-						scales: {
-							yAxes: [
-								{
-									display: false,
-									ticks: {beginAtZero: true},
-								}],
-							xAxes: [{display: false}],
-						},
-						layout: {
-							padding: {top: 2},
-						},
-					};
-				} else if (this.type === 'muted-sparkline') {
-					return {
-						responsive: true,
-						//animation: false,
-						//aspectRatio: this.getAspectRatio(),
-						legend: {display: false},
-						elements: {
-							line: {
-								borderColor: 'rgba(211, 167, 138, 0.2)',
-								borderWidth: 2,
-							},
-							point: {
-								radius: 0,
-							},
-						},
-						tooltips: {enabled: false},
-						scales: {
-							y: {
-                  beginAtZero: true,
-									display: false,
+import Chart from 'chart.js/auto';
```

Comment:
Going forward please make formatting changes in a separate commit (same branch/PR is fine though), it's pretty hard to work out what has changed.

Reply:
My IDE does it automatically whenever I save, but I'll turn that feature off.